README.md

```diff
@@ -219,80 +219,58 @@ The corresponding function in on Tarantool Cartridge router may look like (uses
-    List<SampleUser> getPredefinedUsers();
+    @Query(function = "get_predefined_user", output = TarantoolSerializationType.AUTO)
+    SampleUser predefinedUser();
 }
 ```
 
-```java
-@Tuple
-public class Book {
-  @Id
-  private Integer id;
-  private String name;
-}
+```lua
+function get_users_with_gt_age(age)
+    return crud.select("sample_user", { { ">", "age", age } })
+end
 
-// Here, the space name is "another_space" for all @Query methods
-@Tuple("another_space") 
-public interface BookRepository extends TarantoolRepository<Book, Integer> {
-    
-    // The space name here overrides "another_space" to "test_space" for the current method
-    @Tuple("test_space") 
-    @Query(function = "do_something")
-    Optional<Book> doSomething(Book book);
-}
+function get_predefined_user()
+    return { name = "John", age = 46 }
+end
 ```
+The first function returns users who are older than the specified age.
+In the Repository, we indicate that we expect `TUPLE`, because `crud.select` returns the result in compressed format, flatten tuples.
+The keys for mapping the result into a Java object are obtained by the connector by the class name in returnType or by the specified `@Tuple` above it.
 
-Here the space name is not specified neither in a `@Tuple` annotation on the repository interface, nor on the entity class, so it will be "book_entity" according to the entity class name, turned into "lower snake case".
-But for the "doSomething" method the space name will be overridden to "test_space".
-```java
-@Tuple
-public class BookEntity {
-  @Id
-  private Integer id;
-  private String name;
-}
+The second function returns a ready-made result with keys.
+So `TUPLE` will not work here, because it's not flatten structure, and we specify `AUTO`.
+Mapping happens by keys of table.
 
-public interface BookRepository extends TarantoolRepository<BookEntity, Integer> {
-    
-    // The space name here overrides to "test_space" for the current method
-    @Tuple("test_space") 
-    @Query(function = "do_something")
-    void doSomething(Book book);
-}
+Of course, the word `AUTO` means that we can accept any result, and therefore it can be specified in the first request as well.
+But we specify `TUPLE` because a different stack of converters is used and the conversion is faster.
+For the same reason, the **default output parameter** is `TarantoolSerializationType.TUPLE`.
+
+We can also return primitive types if needed:
+```java
+@Query(function = "get_age_by_name", output = TarantoolSerializationType.AUTO)
+Integer getAgeByName(String name);
 ```
 
 ```lua
-function returning_sample_user_object()
-    return { name = "John", lastName = "Smith" }
-end
-
-function get_predefined_users()
-    return { { name = "John", lastName = "Smith" }, { name = "Sam", lastName = "Smith" } }
+function get_age_by_name(name)
+    local user = crud.get("sample_user", name)
+    if user.rows[1] ~= nil then
+        return user.rows[1][2]
+    end
+    return nil
```

Comment:
That's a very strange example. It either should be additional (explaining that nil values are parsed correctly) or a normal number should be returned.

Reply:
What do you mean by normal number? default?

Reply:
1, 2, 10 -- any integer

Reply:
Yes, you're right, it doesn't work that way. We can add `Optional<Integer>` and test in sources